api/src/lib/str/sanitize.ts

```
@@ -0,0 +1,13 @@
+/**
+ * Sanitize a string a remove non UTF-8 chars
+ */
+export function sanitize(str: string, maxLength = 4096): string {
+  return str
+    .replace(/\u20AC/g, "e") // € -> e
+    .normalize("NFD")
+    .replace(/[\ \.\/]/g, "_")
+    .replace(/([\u0300-\u036f]|[^\w-_\ ])/g, "")
```

Biome (1.9.4)

[error] A character class cannot match a character and a combining character.

A character and a combining character forms a new character. Replace the character class with an alternation.

(lint/suspicious/noMisleadingCharacterClass)